### ISSUE TITLE
Number of epochs not centralized

### ISSUE THREAD

**Issue by csmagnano** [gitter-lab/pathway-localization#2]
Currently we set the number of epochs to run gradient descent for in different places in the code for different experiments, making it difficult to keep track of how many epochs are being run or where it can be set. 

We set it:
[databasePrediction/localizationTuningAx.py  line 34](https://github.com/gitter-lab/pathway-localization/blob/d35125ef26a6103e2e7aeaa3b76104d285260f2e/databasePrediction/localizationTuningAx.py#L34) - This is set to 1000 epochs which we use for tuning. 
[databasePrediction/localizationPyTorchGeo.py line 239](https://github.com/gitter-lab/pathway-localization/blob/d35125ef26a6103e2e7aeaa3b76104d285260f2e/databasePrediction/localizationPyTorchGeo.py#L239) - This is set to 2000 epochs which we use for full training post tuning. 
[caseStudy/trainCaseStudy.py line 258](https://github.com/gitter-lab/pathway-localization/blob/d35125ef26a6103e2e7aeaa3b76104d285260f2e/caseStudy/trainCaseStudy.py#L258) - This is set to 300 epochs which we use for the case study (since the datasets are smaller).
